src/components/player/cue-bar.tsx

```diff
@@ -98,7 +98,7 @@ const NoteCue: React.FC<any> = ({
   const visible =
     player.activeMetadataTrackCues.includes(cue) && !player.seeking
   const startPosition = `${(cue.startTime / duration) * 100}%`
-  const note = JSON.parse(cue.text)
+  const note = cue.text
```

Comment:
not using JSON, which might be something we test for in the future. We **can** just not needed for the current approach.